File: Array/Primes in Array/prime_array.py
def isPrime(n):
    f=0
    for i in range(2,n-1):
        if n%i==0:
            f=1
            break
    if f==1:
        return False
    else:
        return True

a=[]
print("Enter 10 numbers : ",end="")
for i in range(10):
    a.append(int(input()))
for i in range(10):
    if isPrime(a[i]):
        print("{}, ".format(a[i]),end=" ")

File: Array/Primes in Array/prime_array.java
import java.util.*;
class prime_array{
    boolean isPrime(int n){
        int f=0;
        for(int i=2;i<n;i++){
            if(n%i==0){
                f=1;
                break;
            }
        }
        if(f==1)
            return false;
        else
            return true;
    }
    public static void main(String[] args){
        prime_array ob=new prime_array();
        Scanner sc=new Scanner(System.in);
        int a[]=new int[10];
        System.out.print("Enter 10 numbers : ");
        for (int i=0;i<10;i++)
            a[i]=sc.nextInt();
        sc.close();
        System.out.print("Prime numbers : ");
        for(int i=0;i<10;i++){
            if(ob.isPrime(a[i]))
                System.out.print(a[i]+", ");
        }
    }
}